package.json

```diff
@@ -31,47 +31,46 @@
   "homepage": "https://github.com/esbb48/Kizuna#readme",
   "dependencies": {
     "babel-polyfill": "^6.9.1",
-    "grommet": "^0.6.10",
+    "extract-text-webpack-plugin": "^v2.0.0-beta.4",
+    "grommet": "^1.0.1",
```

Comment:
According to our conversation, `` yarn outdated | sed '1,2d;s/\(\S\+\)\s\+\S\+\s\+\S\+\s\+\(\S\+\)\s\+\S\+/\1@\2/;$d' | xargs npm install `` does that. :)